docs/src/tutorials/weak_order_conv_sde.md

```diff
@@ -0,0 +1,41 @@
+# [Using the EnsembleGPUKernel SDE solvers for the expectation of SDEs ](@id sdeweakconv)
+
+Solving the `SDEProblem` using weak methods on multiple trajectories helps to generate the expectation of the stochastic process.
+With the lower overhead of `EnsembleGPUKernel` API, these calculations can be done in parallel on GPU, potentially being faster.
+
+The example below provides a way to calculate the expectation time-series of a linear SDE:
+
+```julia
+using DiffEqGPU, OrdinaryDiffEq, StaticArrays, LinearAlgebra, Statistics
+
+num_trajectories = 10_000
+
+# Defining the Problem
+# dX = pudt + qudW
+u₀ = SA[0.1f0]
+f(u, p, t) = SA[p[1] * u[1]]
+g(u, p, t) = SA[p[2] * u[1]]
+tspan = (0.0f0, 1.0f0)
+p = SA[1.5f0, 0.01f0]
```

Comment:
This noise term is too small to see that the analytical solution is incorrect. Change this to be more differentiating.